open_xdmod/modules/xdmod/regression_tests/post_ingest_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# Tests that cloud ETL works after upgrade + reingestion
+
+BASEDIR=./open_xdmod/modules/xdmod/regression_tests
+REF_DIR=/var/tmp/referencedata
+last_modified_start_date=$(date +'%F %T')
+
+oldCount=$(echo $(mysql -e "SELECT SUM(num_sessions_ended) FROM modw_cloud.cloudfact_by_month WHERE month = 4;") | cut -d ' ' -f 2)
```

Comment:
This can be simplified:
```suggestion
oldCount=$(mysql -N -B -e "SELECT SUM(num_sessions_ended) FROM modw_cloud.cloudfact_by_month WHERE month = 4;")
```